### PR TITLE
in-dexter:0.0.3

### PR DESCRIPTION
**Pull request by RolfBremer**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

Description: Generate hand picked index from marked entries in the document.

### CODE REVIEW

**Comment by istudyatuni** [comment]
There are some extra indents in readme, maybe remove them?

File: packages/preview/in-dexter-0.0.3/README.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+## Importing the Component
+
+To use the index functionality, the component must be available. This
+can be achieved by importing the package `in-dexter` into the project:
+
+Add the following code to the head of the document file(s)
+that want to use the index:
+
+```typ
+  #import "@preview/in-dexter:0.0.3": *
```
Comment (laurmaedje) [inline packages/preview/in-dexter-0.0.3/README.md:38]: there's still some indent here

File: packages/preview/in-dexter-0.0.3/README.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+the word, as well as its location in the document.
+
+== Controlling the Show
+
+At the start of this document, just after the `import` #index[Import] statements, we used
+a `show` rule #index[Show Rule] to define, what we want to see of the index markers in the
+resulting document. This is usually defined in a way, that the markers just show nothing:
+
+```typ
+  // Index-Entry hiding : this rule makes the index entries in the document invisible.
```
Comment (laurmaedje) [inline packages/preview/in-dexter-0.0.3/README.md:63]: and here, and probably in other places

Comment (RolfBremer) [reply packages/preview/in-dexter-0.0.3/README.md]: Ok, removed them all.

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**